programs/ziti-edge-tunnel/windows-scripts.c

```diff
@@ -362,20 +363,140 @@ void remove_all_nrpt_rules() {
     }
 }
 
+void chunked_remove_and_add_nrpt_rules(uv_loop_t *ziti_loop, hostname_list_t *hostnames, const char* dns_ip) {
+    char* script = calloc(MAX_POWERSHELL_SCRIPT_LEN, sizeof(char));
```

Comment:
I tried to change the declaration to char array, but the exec_process is not executing this function. In my IDE I see junk characters, because the memory is not allocated for this script variable properly, when it is passed to the powershell

Reply:
How exactly do you declare and initialize the array? If it is being done as in my previous comment and you see garbage characters then I'd suspect an underlying problem with the stack.

Reply:
I had seen garbage characters when I ran it in clion once. Now I dont see it anymore. I changed the char pointer to char array.